docs/ClusterSetup.md

```diff
@@ -183,6 +183,9 @@ ledgerDirectories=data/bookkeeper/ledgers
 
 # Point to local ZK quorum
 zkServers=zk1.example.com:2181,zk2.example.com:2181,zk3.example.com:2181
+
+# Change the ledger manager type
+ledgerManagerType=hierarchical
```

Comment:
That might be the case, indeed. However, these changes are absolute minimum to make it work without having to modify existing files (i.e. create bare minimum viable config). When using configuration management in any form, it might be easier to just apply the minimum config.